README.md

```diff
@@ -1,54 +1,87 @@
 # Mixlib::ShellOut
-Provides a simplified interface to shelling out yet still collecting both
-standard out and standard error and providing full control over environment,
-working directory, uid, gid, etc.
+[![Build Status Master](https://travis-ci.org/chef/mixlib-shellout.svg?branch=master)](https://travis-ci.org/chef/mixlib-shellout) [![Build Status Master](https://ci.appveyor.com/api/projects/status/github/chef/mixlib-shellout?branch=master&svg=true&passingText=master%20-%20Ok&pendingText=master%20-%20Pending&failingText=master%20-%20Failing)](https://ci.appveyor.com/project/Chef/mixlib-shellout/branch/master) [![Gem Version](https://badge.fury.io/rb/mixlib-shellout.svg)](https://badge.fury.io/rb/mixlib-shellout)
+
+Provides a simplified interface to shelling out while still collecting both standard out and standard error and providing full control over environment, working directory, uid, gid, etc.
 
 No means for passing input to the subprocess is provided.
 
 ## Example
+### Simple Shellout
 Invoke find(1) to search for .rb files:
 
-      find = Mixlib::ShellOut.new("find . -name '*.rb'")
-      find.run_command
+```ruby
+  require 'mixlib/shellout'
+  find = Mixlib::ShellOut.new("find . -name '*.rb'")
+  find.run_command
+```
 
 If all went well, the results are on `stdout`
 
-      puts find.stdout
+```ruby
+  puts find.stdout
+```
 
 `find(1)` prints diagnostic info to STDERR:
 
-      puts "error messages" + find.stderr
+```ruby
+  puts "error messages" + find.stderr
+```
 
 Raise an exception if it didn't exit with 0
 
-      find.error!
+```ruby
+  find.error!
+```
+
+### Advanced Shellout
+In addition to the command to run there are other options that can be set to change the shellout behavior. The complete list of options can be found here: https://github.com/chef/mixlib-shellout/blob/master/lib/mixlib/shellout.rb
```

Comment:
Might be worth calling out the behavior of `ENV` specifically? I agree with your reasoning for delegating to the code that has the exhaustive list, but env seems important for a lot of usage and it's behavior is a little tricky.